src/display/annotation_layer.js

```diff
@@ -558,6 +559,14 @@ class AnnotationElement {
     }
     return fields;
   }
+
+  show() {
+    this.container.hidden = false;
```

Comment:
*This probably doesn't matter in practice, but nonetheless:*
To prevent a future bug, since in the general case there's no *guarantee* that every `AnnotationElement`-instance actually has a `this.container` property.
```suggestion
    if (this.container) {
      this.container.hidden = false;
    }
```